internal/packages/packages.go

```diff
@@ -110,7 +110,10 @@ type PackageManifest struct {
 	Owner           Owner            `config:"owner" json:"owner" yaml:"owner"`
 	Description     string           `config:"description" json:"description" yaml:"description"`
 	License         string           `config:"license" json:"license" yaml:"license"`
-	Categories      []string         `config:"categories" json:"categories" yaml:"categories"`
+	Source          struct {
+		License string `config:"license" json:"license" yaml:"license"`
```

Comment:
Do you plan to deprecate the license field in this PR?

Reply:
It is [deprecated in the package-spec](https://github.com/elastic/package-spec/pull/355/files#diff-654fe06534f569b9f336e21e2cac527bfb3d15a9d19cf6f3995915ed2445f208R35).
I think we are not doing anything with deprecations in `elastic-package`, do you have a proposal for something that we could do here?

Reply:
I would drop a comment in the source in this place as both fields are modeled. Frankly speaking, it's another argument for switching to a common place with business/model structures.

Reply:
> Frankly speaking, it's another argument for switching to a common place with business/model structures.

Yep, another +1 for something like https://github.com/elastic/package-spec/issues/322.